Apps/Common/src/Auditing/AuditLogger.cs

```diff
@@ -57,10 +57,17 @@ public void PopulateWithHttpContext(HttpContext context, AuditEvent auditEvent)
             auditEvent.ApplicationType = GetApplicationType();
             auditEvent.TransactionResultCode = GetTransactionResultType(context.Response.StatusCode);
 
-            auditEvent.ApplicationSubject = hdid ?? subjectPhn ?? subjectQuery;
+            // Check if Hdid is in route values
+            RouteValueDictionary routeValues = context.Request.RouteValues;
+            string? routeHdid = routeValues["Hdid"] as string;
+
+            // Check if Hdid is in query parameters
+            context.Request.Query.TryGetValue("Hdid", out StringValues queryHdidParameter);
```

Comment:
I had to double-check to verify these retrievals were both case-insensitive, but they are.

Reply:
Route values are case sensitive but query parameters are not.